docs/reference/commandline/service_create.md

```diff
@@ -474,6 +474,42 @@ accessible at the target port on every node regardless if there is a task for
 the service running on the node. For more information refer to
 [Use swarm mode routing mesh](https://docs.docker.com/engine/swarm/ingress/).
 
+### Templating
```

Comment:
Do we want to call it "Templating" or maybe something like "Create services using templates"?